apps/map-viewer/src/app/app.module.ts

```diff
@@ -31,15 +31,7 @@ export const metaReducers: MetaReducer<any>[] = !environment.production
     StoreDevtoolsModule.instrument(),
     EffectsModule.forRoot(),
   ],
-  providers: [
-    {
-      provide: Configuration,
-      useFactory: () =>
-        new Configuration({
-          basePath: '/geonetwork/srv/api',
-        }),
-    },
-  ],
+  providers: [provideRepositoryUrl('/geonetwork/srv/api')],
```

Comment:
Why does the `map-viewer` not use this value from the config?

Reply:
the map viewer doesn't use the app config system yet, it just has a hardcoded value here; I would say it's intentional until we have more budget to improve this app